### DISCUSSION TITLE
changing the color of a single edge or a subset of edges

### DISCUSSION THREAD

**Discussion by CTMMS** [jacomyal/sigma.js#1465] [Q&A]
Is changing the color of a single edge possible? 
Whenever I try changing a single edge's attributes, the change is rendered for all of them.

**Comment by jacomyal** [comment]
You can read the documentation on [Customizing graph appearance](https://www.sigmajs.org/docs/advanced/customization), or the [Events](https://www.sigmajs.org/storybook/?path=/story/events--story) story, where hovering an edge modifies only its color.

Though, we can't really help you without actual code. Could you provide some code to reproduce your issue?

**Reply by jacomyal** [reply]
Hi @CTMMS,

Any news on some more information to help me help you? Or maybe the code from the two stories I linked in my previous message did help you solving your issue?

**Reply by CTMMS** [reply]
Although I was very grateful that you took the time to respond, what I wanted was reassurance that changing the attributes of a single edge instead of all of them (without implementing hover events) was supposed to be possible in order to know whether I should keep trying to fix it or move on.

Being told to read the documentation and examples I had already read multiple times before reaching out and that I had worded my question wrong made me feel too uncomfortable to keep trying to get help, and I ended up fixing my problem on my own. Thank you for checking in.
